src/api/app/jobs/staging_project_copy_job.rb

```diff
@@ -2,6 +2,14 @@ class StagingProjectCopyJob < ApplicationJob
   def perform(staging_workflow_project_name, original_staging_project_name, staging_project_copy_name)
     staging_workflow_project = Project.find_by!(name: staging_workflow_project_name)
     original_staging_project = staging_workflow_project.staging.staging_projects.find_by!(name: original_staging_project_name)
-    original_staging_project.copy(staging_project_copy_name)
+
+    begin
```

Comment:
There is a built-in way to rescue from exception in ActiveJob. It is `rescue_from` (really original, I know!!!).

https://edgeguides.rubyonrails.org/active_job_basics.html#exceptions

Reply:
Oh nice! I'll change it